### PR TITLE
new icon: swagger (original, original-wordmark, plain, plain-wordmark)

### LINKED ISSUE #903: Icon Request: Swagger
### About the icon

Swagger is a tool to design and document APIs

### Links and sources

https://swaggerhub.com

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by yeyeto2788**
## Double check these details before you open a PR

- [X] PR does not match another non-stale PR currently opened
- [X] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [X] PR's base is the `develop` branch.
- [X] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [X] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [X] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

**This PR closes #903**

### CODE REVIEW

File: devicon.json [hunk 1]
```diff
@@ -7756,6 +7756,26 @@
...
+           {
+           "base": "original",
+           "alias": "original-wordmark"
+           },
+           {
+           "base": "plain",
+           "alias": "plain-wordmark"
+           }
+       ]
+    },
```
Comment (PantheraRed) [inline devicon.json:7778]: I'd suggest some changes.

```suggestion
    {
       "name": "swagger",
       "altnames": [],
       "tags": [
           "development",
           "api",
           "documentation",
           "tool"
       ],
       "versions": {
           "svg": [
               "original",
               "plain"
           ],
           "font": [
               "original",
               "plain"
           ]
       },
       "color": "#173647",
       "aliases": [
           {
               "base": "original",
               "alias": "original-wordmark"
           },
           {
               "base": "plain",
               "alias": "plain-wordmark"
           }
       ]
    },
```

**Comment by PantheraRed** [comment]
@yeyeto2788, I think it would be a good idea to separate icon from its wordmark version and call it the `original` version. Also fonts made out of `original` and `plain` versions are very small and multi-colored. I have made a few changes and new versions. Please have a look and give your opinion on them.

# swagger-original

swagger-original

# swagger-original-wordmark

swagger-original-wordmark

# swagger-plain

swagger-plain

# swagger-plain-wordmark

swagger-plain-wordmark

**Comment by yeyeto2788** [comment]
@PantheraRed that is fine for me but then the original wouldn't be the original as the real one has the wording on it.

I regards of your versions, they do look great considering that I was using the blueish color as the predominant one but definitely the green one pops a lot.

@Snailedlt, if you still want me to change and agree with @PantheraRed it just let me know. 

Best.

**Comment by lunatic-fox** [comment]
Hello there!

The main color of this icon is `#173647` and I agree with @PantheraRed in `original` and `original-wordmark` versions.

`plain` and `plain-wordmark` versions can be done from the current `plain` version, but I prefer the idea of @PantheraRed since variations of this color `#85ea2d` are more frequent than variations of `#173647` on their [website](https://swagger.io/), suggesting that it should be the main theme color related to this technology.

The `original` version is fine since they're using it as [favicon](https://static1.smartbear.co/swagger/media/assets/swagger_fav.png) and if you look at the bottom of their [website](https://swagger.io/) they're also using this version of the icon. And `original-wordmark` is similar to `original` version, but with the technology name.

If you choose the icons with `#85ea2d` color, remember to change `color` property before make the new commit. We also have to remove the `<clipPath>` from `plain` and `plain-wordmark*`.
> *\* To be done.*

---
> ### 💡 General tip
> `plain`, `plain-wordmark`, `line` and `line-versions` don't really need a fill color, as long as those versions receive the fill color from `devicon.json` `color` property. The only exception happens when some version is an alias from `original` or `original-wordmark` versions.

**Comment by Snailedlt** [comment]
@yeyeto2788 I agree with what @PantheraRed and @lunatic-fox suggested, so go ahead 😊

**Comment by PantheraRed** [comment]
@lunatic-fox,
> If you choose the icons with `#85ea2d` color, remember to change `color` property before make the new commit. We also have to remove the `<clipPath>` from `plain` and `plain-wordmark*`.

All the SVGs in this https://github.com/devicons/devicon/pull/1561#issuecomment-1333681625 are already optimized and don't have `<clipPath>` attributes.

**Comment by lunatic-fox** [comment]
> @lunatic-fox,
> 
> > If you choose the icons with `#85ea2d` color, remember to change `color` property before make the new commit. We also have to remove the `<clipPath>` from `plain` and `plain-wordmark*`.
> 
> All the SVGs in this [#1561 (comment)](https://github.com/devicons/devicon/pull/1561#issuecomment-1333681625) are already optimized and don't have `<clipPath>` attributes.

I was talking based on @yeyeto2788 icons, but that's great, since the icons that you made are just ready-to-go. 🤩

**Comment by yeyeto2788** [comment]
Hey guys! Sorry for the delay.

Changes applied, thanks @PantheraRed for providing the icons and @Snailedlt and @lunatic-fox for your quick reply.

File: devicon.json [hunk 2]
```diff
@@ -7758,6 +7758,29 @@
...
         "aliases": []
     },
+    {
+        "name": "swagger",
+        "altnames": [
+            "Swagger"
+        ],
+        "tags": [
+            "development"
+        ],
```
Comment (lunatic-fox) [inline devicon.json:7768]: Some suggestions of tag.
```suggestion
        "tags": [
            "development",
            "software",
            "tool"
        ],
```

**Review by lunatic-fox — CHANGES_REQUESTED** [review]
Made some commits to fix the `devicon.json` structure and a little resize to the icons.

@yeyeto2788, I think you should check out for some linter on the IDE that you're using to edit `devicon.json`

**Review by lunatic-fox — APPROVED** [review]
Approved! ✔
Thank you for this contribution! 🚀

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**